src/Tribe/Views/V2/Hooks.php

```diff
@@ -881,6 +884,10 @@ public function filter_customizer_sections( $sections, $customizer ) {
 		return $this->container->make( Customizer::class )->filter_sections( $sections, $customizer );
 	}
 
+	public function action_include_global_elements_settings( $section, $manager, $customizer ) {
```

Comment:
:no_entry_sign: Missing doc comment for function action_include_global_elements_settings()

<sub>hash: 7b79994eeff24447631e540381c690fa</sub>